### DISCUSSION TITLE
Ability to control tab height from settings just like font size

### DISCUSSION THREAD

**Discussion by bgrr74** [kinghat/tabbed-card#40] [Feature Requests]
Is it possible to change the height of the tabs ?

After changing the font to a smaller font with the option the tab height stays the same.

I would like tabs with less height so i can fit the tabs nicely in my layout / design

Greetings Barry

**Comment by kinghat** [comment]
i dont think i will add much outside of the underlying tabs web components api to explicitly change it. unless there is a lot of support for it. you can already see there are a lot of one off feature requests and its not really practical to handle them all. not saying never but it most likely wont be an explicit option but you might be able to pass your own css.

**Comment by kinghat** [comment]
> Is it possible to change the height of the tabs ?
> 
> After changing the font to a smaller font with the option the tab height stays the same.
> 
> I would like tabs with less height so i can fit the tabs nicely in my layout / design
> 
> Greetings Barry

sorry, i also forgot, in the [styling](https://github.com/kinghat/tabbed-card#styling) section of the readme, theres a link to the [tabs api](https://github.com/material-components/material-web/blob/mwc/packages/tab/README.md#css-custom-properties) which shows you what can be explicitly configured. you can set the tab height `--mdc-tab-height: 24px` for instance.

**Reply by bgrr74** [reply]
Just returned here to post the same solution and saw you already posted it an hout ago :-)

Working perfect for me.

  - type: custom:tabbed-card
    styles:
      '--mdc-typography-button-font-size': 12px
      '--mdc-tab-height': 25px
